base/applications/cmdutils/where/lang/en-US.rc

```diff
@@ -29,11 +29,11 @@ NOTE: This tool returns an error level of 0 if the search was successful,\n\
       1 if the file was not found, and 2 if there was an error.\n\
 \n\
 Example:\n\
-    WHERE myfile*.exe\n\
-    WHERE /F /T mspaint\n\
-    WHERE $WINDIR:notepad myfile???\n\
-    WHERE C:\\ReactOS;C:\\ReactOS\\system32:exp*.exe\n\
-    WHERE /R ""C:\\Program Files"" *.dll\n"
+    where myfile*.exe\n\
+    where /F /T mspaint\n\
+    where $WINDIR:notepad myfile???\n\
+    where C:\\ReactOS;C:\\ReactOS\\system32:exp*.exe\n\
+    where /R ""C:\\Program Files"" *.dll\n"
```

Comment:
🤔 Not sure it makes really sense to have a different case there, while the command name above was in upper case (just my opinion).

Reply:
Reverted to uppercase.